pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[build-system]
+requires = ["hatchling", "hatch-requirements-txt", "hatch-nodejs-version"]
+build-backend = "hatchling.build"
+
+[project]
+name = "mkdocs-material"
+dynamic = [
+  "version",
+  "dependencies",
+  "license",
+  "description",
+  "authors",
+  "keywords",
+  "urls",
+]
+readme = "README.md"
+requires-python = ">=3.7"
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Environment :: Web Environment",
+  "License :: OSI Approved :: MIT License",
+  "Programming Language :: JavaScript",
+  "Programming Language :: Python",
+  "Topic :: Documentation",
+  "Topic :: Software Development :: Documentation",
+  "Topic :: Text Processing :: Markup :: HTML",
+]
+
+[project.entry-points."mkdocs.plugins"]
+search = "material.plugins.search.plugin:SearchPlugin"
```

Comment:
Could you please update the PR to include the `social` plugin? It was released as part of 8.5.0.

Reply:
Done!